catalog/iam-foundation/networking.yaml

```diff
@@ -23,9 +23,9 @@ spec:
   resourceRef:
     apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
     kind: Organization
-    external: "123456789012" # kpt-set: ${org-id}
+    external: sample-org-id
   role: roles/compute.networkAdmin
-  member: group:gcp-network-admins@example.com # kpt-set: group:${group-network-admins}
+  member: group:gcp-network-admins@sample-domain.com
```

Comment:
One reason we chose "example.com" was that it's reserved for that purpose:
https://en.wikipedia.org/wiki/Example.com

Reply:
Good point - should we use `example` as our standard placeholder instead of `sample`?

Reply:
Wow, thank you @bgrant0607 ! I didn't know this. I'll change them back to `example.com`.

@morgante We can treat the well-known conventions specifically and continue our own convention `sample-<resourceType>`. But either sample or example works for me.

Reply:
I'd like to standardize on one value as a convention and since `example.com` is used for domains we can use `example-<resource-type>` for everything.

Reply:
Changed `sample` to `example`. Here and other PRs.